src/Components/Footer.tsx

```diff
@@ -0,0 +1,83 @@
+import classNames from 'classnames';
+import React from 'react';
+import { FilterType } from '../types/FilterType';
+import { Todo } from '../types/Todo';
+
+type Props = {
+  todos: Todo[],
+  filterType: FilterType,
+  setFilterType: (filterType: FilterType) => void,
+  completedTodos: Todo[],
+  onDelete: (todoId: number) => void,
+};
+
+export const Footer: React.FC<Props> = ({
+  todos,
+  filterType,
+  setFilterType,
+  completedTodos,
+  onDelete,
+}) => {
+  const activeTodos = todos.filter(todo => !todo.completed).length;
+  const allCompletedTodos = completedTodos.map(completeTodo => completeTodo.id);
+
+  const deleteAllCompletedTodos = (allCompletedTodo: number[]) => {
+    allCompletedTodo.forEach(todoId => onDelete(todoId));
+  };
+
+  return (
+    <footer className="todoapp__footer">
+      <span className="todo-count">
+        {activeTodos === 1
+          ? '1 item left'
+          : `${activeTodos} items left`}
```

Comment:
one of the possible improvements 
```suggestion
        {`${activeTodos} item${activeTodos === 1 ? '' : 's'} left`}
```